builds/configuring-build-strategies.adoc

```diff
@@ -0,0 +1,38 @@
+:_content-type: ASSEMBLY
+[id="configuring-build-strategies"]
+= Configuring build strategies
+include::_attributes/common-attributes.adoc[]
+:context: configuring-build-strategies
+
+toc::[]
+
+
+A `BuildStrategy` or `ClusterBuildStrategy` custom resource (CR) helps you to define strategy parameters, system parameters, steps resource definition, annotations, and volumes to configure a build strategy. A `BuildStrategy` resource is available for use within a namespace, and a `ClusterBuildStrategy` resource is available for use throughout the cluster.
```

Comment:
```suggestion
A `BuildStrategy` or `ClusterBuildStrategy` custom resource (CR) helps you to define strategy parameters, system parameters, steps resource definitions, annotations, and volumes to configure a build strategy. A `BuildStrategy` resource is available for use within a namespace, and a `ClusterBuildStrategy` resource is available for use throughout the cluster.
```

Or "step resources definitions"/"a step resource definition".

Also, could "to configure a build strategy" be "for build strategy configuration"?

Reply:
I would like to highlight the action part like you can use these items to configure a feature. So, going with "to configure a build strategy".